hana/get_platform.sls

```diff
@@ -0,0 +1,18 @@
+#!py
+import platform
+
+PLATFORMS = {
+    'x86_64': 'HDB_CLIENT_LINUX_X86_64',
+    'ppc64': 'HDB_CLIENT_LINUX_PPC64',
+    'ppc64le': 'HDB_CLIENT_LINUX_PPC64LE'
+}
```

Comment:
Hi @arbulu89 you added the HDB_CLIENT_ prefix to PLATFORMS.
It works for our scenario but maybe this is not expected result if just fetching the platform type

Reply:
Hi @Simranpal I don't really understand your point.
What should we put there in your opinion?
As far as I have seen in all of our SAP installation folder, these are the names used by SAP for these platforms

Reply:
Hi @arbulu89 what I mean is get_platform.sls should return the platform type x86_64/ ppc64 etc. Currently, this returns the directory name for HDB Client (thats what is required now).

I meant to correct the names, I think here filename or the PLATFORMS variable could be renamed to something else, like HDB_CLIENT TYPE

Reply:
I think  that `PLATFORMS` is ok, imho.

If we want to make more precision: `HDB_CLIENT_PLATFORMS`  imho

`HDB_CLIENT_TYPE` type is to generic word, and one could understand as type of client (http/whatever protocols).

Reply:
So, the discussion is about the naming, right?
Ok, I would think other way of renaming the sls file and the code.
Besides that, what do you think about the functionality @Simranpal ?

Reply:
The functionality looks good @arbulu89

Reply:
@arbulu89 I think we are mixing platform with HANA components here IMO.  The directory name determines the HANA component that it has inside + the platform where it was built. as HANA components we have `HANA Client`, `HANA Server`, `HANA XServer` and many others, but currently we install only some of these components. So, in that sense, I would split the directory component name, from the arch that composes the name, so we can reuse it in the future for the new components that we eventually will install. (they will come for S/4 HANA for exmple).

Reply:
Ok. So I will change the code to only return the platform (formatted to capital letters) and create the path to the client with that.